components/Admin/Admin.js

```diff
@@ -20,7 +25,7 @@ class Admin extends React.Component {
   componentWillMount() {
     fetchLocations()
       .then(locations => {
-        this.setState({ locations })
+        this.setState({ loading: false, locations })
```

Comment:
You can still have

```javascript
const loading = this.state.locations == null ? true : false;
```

But instead it's derived from exactly what it's referencing!

P.S. this _is_ nitpicky and unnecessary. But it'll be a powerful concept when you help we work on the elm-calendar. So you can choose to change it or not, won't stop me from 👍